File: tlg_android/src/ch/bailu/tlg_android/NoDrawContext.java



package ch.bailu.tlg_android;

import android.content.Context;
import ch.bailu.tlg.TlgPoint;
import ch.bailu.tlg.TlgRectangle;

public class NoDrawContext extends AndroidContext {

	public NoDrawContext(Context c) {
		super(c);
	}

	
	@Override
	public void drawLine(int color, TlgPoint p1, TlgPoint p2) {
	}

	@Override
	public void drawFilledRectangle(int color, TlgRectangle rect) {
	}

	@Override
	public void drawText(int color, TlgRectangle rect, String text) {
	}


	@Override
	public void unlockCanvas() {
	}


	@Override
	public void onNewHighscore() {
		
	}
}

File: tlg_android/src/main/java/ch/bailu/tlg_android/context/NoDrawContext.kt
package ch.bailu.tlg_android.context

import android.content.Context
import ch.bailu.tlg.TlgPoint
import ch.bailu.tlg.TlgRectangle

open class NoDrawContext(c: Context) : AndroidContext(c) {
    override fun drawLine(color: Int, p1: TlgPoint, p2: TlgPoint) {}
    override fun drawFilledRectangle(color: Int, rect: TlgRectangle) {}
    override fun drawText(color: Int, rect: TlgRectangle, text: String) {}
    override fun unlockCanvas() {}
    override fun onNewHighscore() {}
}
